src/data/messaging.js

```diff
@@ -41,3 +52,26 @@ export async function saw(conversationId) {
     console.error(`can't update last viewed for ${conversationId} at this time so messages will continue to show as unread, absorbing exception: ${e}`)
   }
 }
+
+export async function send(message, conversation) {
+  try {
+    const { createMessage } = await sendMessage(message, conversation)
+    const updatedConversation = createMessage.thread
+
+    conversations.update(currentConversations => {
+      const i = currentConversations.findIndex(({ id }) => id === updatedConversation.id)
+      if (i >= 0) {
+        currentConversations[i] = updatedConversation
+      } else {
+        currentConversations.push(updatedConversation)
```

Comment:
May want to stick the new conversation at the beginning of the list of conversations